diff --git a/node_modules/@storybook/addon-actions/node_modules/uuid/dist/rng-browser.js b/node_modules/@storybook/addon-actions/node_modules/uuid/dist/rng-browser.js
index d3cc308..1548598 100644
--- a/node_modules/@storybook/addon-actions/node_modules/uuid/dist/rng-browser.js
+++ b/node_modules/@storybook/addon-actions/node_modules/uuid/dist/rng-browser.js
@@ -14,7 +14,12 @@ const rnds8 = new Uint8Array(16);
 
 function rng() {
   if (!getRandomValues) {
-    throw new Error('crypto.getRandomValues() not supported. See https://github.com/uuidjs/uuid#getrandomvalues-not-supported');
+    var l = rnds8.length
+    while (l--) {
+      rnds8[l] = Math.floor(Math.random() * 256)
+    }
+    return rnds8
+    // throw new Error('crypto.getRandomValues() not supported. See https://github.com/uuidjs/uuid#getrandomvalues-not-supported');
   }
 
   return getRandomValues(rnds8);
