diff --git a/node_modules/uuid/dist/rng-browser.js b/node_modules/uuid/dist/rng-browser.js
index 91faeae..a074941 100644
--- a/node_modules/uuid/dist/rng-browser.js
+++ b/node_modules/uuid/dist/rng-browser.js
@@ -18,7 +18,12 @@ function rng() {
     getRandomValues = typeof crypto !== 'undefined' && crypto.getRandomValues && crypto.getRandomValues.bind(crypto) || typeof msCrypto !== 'undefined' && typeof msCrypto.getRandomValues === 'function' && msCrypto.getRandomValues.bind(msCrypto);
 
     if (!getRandomValues) {
-      throw new Error('crypto.getRandomValues() not supported. See https://github.com/uuidjs/uuid#getrandomvalues-not-supported');
+      var l = rnds8.length
+      while (l--) {
+        rnds8[l] = Math.floor(Math.random() * 256)
+      }
+      return rnds8
+      // throw new Error('crypto.getRandomValues() not supported. See https://github.com/uuidjs/uuid#getrandomvalues-not-supported');
     }
   }
 
